index.html

```diff
@@ -0,0 +1,245 @@
+     	width: 50%;
+  		padding: 10px;
+  		height: 100%; 
+	}
+
+	.row:after {
+  		content: "";
+  		display: table;
+  	}
+
+  .vehicletrackimage{
+  		display: block;
+  		margin-left: auto;
+ 		  margin-right: auto;
+  		width: 20%;
+  		height: 20%;
+  	}
+
+  .vehiclelockimage{
+  		display: block;
+  		margin-left: auto;
+ 		  margin-right: auto;
+ 		  width: 30%;
+ 		  height: 20%;
+  	}
+
+  .livealertimage{
+  		display: block;
+  		margin-left: auto;
+ 		  margin-right: auto;
+ 		  width: 20%;
+  		height: 20%;
+  	}
+  	.vehiclehealthimage{
+  		display: block;
+  		margin-left: auto;
+ 		  margin-right: auto;
+ 		  width: 20%;
+  		height: 20%;
+  	}
+  	.headinternal{
+  		text-align: center;
+  		font-size: 16px;
+  		font-family: 'Raleway';
+  		font-weight: 1000;
+  		font-style: normal;
+  		letter-spacing: 0.5px;	
+  	}
+  	.description{
+  		text-align: center;
+  		
+  		line-height: 20px;
+  		font-size: 15px;
+  		font-style:normal;
+  		color: #A3A3A3;
+  		font-family: 'Raleway';
+  		letter-spacing: 0.5px;
+  	}
+  	h4{
+  		font-family: 'Raleway';
+  		font-weight: bold 800;
+  		color: #323232;
+
+  	}
+  	.formbox{
+  		position: relative;
+  		width: 80%;
+  		height: 80%;
+  		margin: 50px;
+  		margin-left: 100px;
+  		margin-top: 30px;
+  		background-color: white;
+  		border:20px solid ;
+  	}
+  	.formint{
+  		margin: 30px;
+  		margin-top: 20px;
+  	}
+  	.inputd{
+  		padding:12px;
+  		width: 80%;
+  		margin-bottom: 20px;
+  		border-radius: 2px;
+  		margin-left: 33px;
+  		border:2px solid #EFEFEF;
+  		color: grey;
+  	  font-weight: bold;
+  		letter-spacing: -0.5px;
+  		font-family: 'Raleway';
+  		font-size: 14px;
+  	}
+  	.loginform {
+  		top: 30px;
+      position: relative;
+      width: 30%;
+      height: 420px;
+      margin-left: 80px;]
+      color: #fff;
+      background: #fff;
+      justify-content: center;
+	}
+	.loginform::after {
+  		content: '';
+ 		  background: linear-gradient( white,grey);
+  		z-index: -1;
+	}
+	.loginform::before {
+  		content: '';
+  		position: absolute;
+  		top: 0;
+  		left: 0;
+  		right: 0;
+  		bottom: 0;
+  		background: linear-gradient( white,grey,grey);
+  		z-index: -2;
+  		filter: blur(20px);
+	}
+	label{
+		color: black;
+		font-size: 13px;
+		padding-bottom: 10px;
+		font-weight: bold;
+		font-family: 'Raleway'
+	}
+	.btn{
+    padding: 10px;
+    width: 25%;
+    margin-right: 5px;
+    font-family: 'Raleway';
+    border:2px solid #EFEFEF;
+    background-image: linear-gradient(0deg,#FFF,#FFF);
+    font-weight: bold;
+    color: grey !important;
+	}
+
+	.startbtn{
+		background-image: linear-gradient(0deg,#1EACB2,#1EACB2);
+		padding: 15px;
+		color: white!important;
+		width: 80%;
+		border:none;
+		border-radius: 6px;
+		font-size: 16px;
+		font-weight: bold;
+		letter-spacing: 0.5px;
+	}
+
+	.callus{
+		margin-left:93px;
+		font-weight: bold;
+		font-size: 12px;
+		color: grey;
+		margin-top: 25px;
+		font-family: 'Roboto';

```

Comment:
is this roboto in design?

Reply:
Yes sir. I checked it again. It's roboto

Reply:
sir I guess it's roboto.